### PR TITLE
feat: KPI-90 reorder layout

### PR DESCRIPTION
**Pull request by dsgarcia8**
## Jira Link

[Go to Jira](https://kpinetwork.atlassian.net/browse/KPI-90)

## Types of changes

- [ ] Hotfix (Bug detected)
- [x] Feature (Issue from an Active Sprint)

## Checklist:

- [ ] Unit Test.
- [ ] Coverage.

### CODE REVIEW

File: src/views/CompanyView/CompanyView.jsx [hunk 1]
```diff
@@ -27,9 +30,12 @@ export function CompanyView ({ params }) {
         </Grid>
       </Grid>
       <Grid container>
-          <Grid item xs={12} sm={8} lg={12} xl={12}><ReportRuleGraph ruleOf40={ruleOf40}/></Grid>
-          <Grid item xs={12} sm={4} lg={6} xl={6}><DesciptionCard description={description} isLoading={isLoading}/></Grid>
-          <Grid item xs={12} sm={4} lg={6} xl={6}><CompanyCard financialProfile={financialProfile} isLoading={isLoading}/></Grid>
+          <Grid item xs={12} sm={8} lg={6}><ReportRuleGraph ruleOf40={ruleOf40}/></Grid>
+          <Grid item xs={12} sm={4} lg={6}><Filter setFilters={setFilters} fillFilters={false} filters={filters}/></Grid>
```
Comment (kellycastrof) [inline src/views/CompanyView/CompanyView.jsx:34]: I think we can move Filter in a different folder, in global components.
Also, we need to add Year select in all pages.

File: src/views/UniverseView/Components/Filter/Filter.jsx [hunk 2]
```diff
@@ -16,21 +16,21 @@ export function Filter ({ setFilters, fillFilters, filters }) {
...
-            <Grid item xs={12} sm={10} lg={6}>
+            <Grid item xs={12} sm={10} md ={6} lg={4} xl={4} style={{marginTop: 20}}>
                <Size setFilters={setFilters}fillFilters={fillFilters} selectedList={size}/>
             </Grid>
-            <Grid item xs={12} sm={10} lg={6}>
+            <Grid item xs={12} sm={10} md ={6} lg={6} xl={6} style={{marginTop: 50}}>
                 <Growth setFilters={setFilters} fillFilters={fillFilters} selectedList={growth}/>
             </Grid>
-            <Grid item xs={12} sm={10} lg={12}>
+            <Grid item xs={12} sm={10} md ={6} lg={12} xl={10} style={{marginTop: 20}}>
```
Comment (kellycastrof) [inline src/views/UniverseView/Components/Filter/Filter.jsx:33]: Can you reduce the space, so we can have in the same row all posible filters?

**Review by kellycastrof — CHANGES_REQUESTED** [review]

File: src/views/CompanyView/CompanyView.jsx (hunk 1 above)
Comment (dsgarcia8) [reply src/views/CompanyView/CompanyView.jsx]: solved comment, filter and header information to select year has been moved to global components

File: src/views/UniverseView/Components/Filter/Filter.jsx (hunk 2 above)
Comment (dsgarcia8) [reply src/views/UniverseView/Components/Filter/Filter.jsx]: solved comment

**Review by kellycastrof — APPROVED** [review]

**Pull request merged**